alphagov/paas-cf
# [#157556295] Update paas-rds-broker for cloudwatch

https://www.pivotaltracker.com/story/show/157556295

What
----

We want to get cloudwatch metrics implemented on:

https://github.com/alphagov/paas-rds-broker/pull/81

Related
-------

 - https://github.com/alphagov/paas-aws-account-wide-terraform/pull/127
 - https://github.com/alphagov/paas-rds-metric-collector/pull/2
 - https://github.com/alphagov/paas-rds-broker/pull/81
 - https://github.com/alphagov/paas-rds-broker-boshrelease/pull/59

How to review
-------------

 - Review https://github.com/alphagov/paas-rds-broker/pull/81
 - Deploy this.
 - Check you get the cpu metric as explained in https://github.com/alphagov/paas-cf/pull/1385

Before merge
--------------

After these  PRs were merge, update the wip commit with the final release

 - https://github.com/alphagov/paas-rds-broker/pull/81
 - https://github.com/alphagov/paas-rds-broker-boshrelease/pull/59


Who can review
--------------

Anyone but @keymon or @paroxp
